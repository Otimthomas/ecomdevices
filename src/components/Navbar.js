import React from "react";
import styled from "styled-components";
import { Link } from "react-router-dom";

function Navbar() {
  return (
    <Wrapper>
      <div className="header">
        <p className="logo">LOGO</p>
        <ul>
          <li>
            <Link to="/">Home</Link>
          </li>
          <li>
            <Link to="/about">About</Link>
          </li>
          <li>
            <Link to="/contact">Contact</Link>
          </li>
        </ul>
      </div>
      <hr />
    </Wrapper>
  );
}

export default Navbar;

const Wrapper = styled.nav`
  /* background-color: orangered; */
  padding: 15px 80px;
  margin: 0;

  .header {
    display: flex;
    justify-content: space-between;
  }

  .logo {
    /* background-color: aquamarine; */
    display: flex;
    align-items: center;
  }

  ul {
    /* background-color: lightblue; */
    display: flex;
  }

  li {
    list-style: none;
  }

  li a {
    text-decoration: none;
    font-size: 20px;
    display: inline-block;
    color: rgb(51, 51, 51);
  }

  li:not(:first-child) {
    margin-left: 50px;
  }
`;
