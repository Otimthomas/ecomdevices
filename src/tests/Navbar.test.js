import Navbar from "../components/Navbar";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

describe("<Navbar />", () => {
  it("matches the snapshot", () => {
    let tree = renderer
      .create(
        <MemoryRouter>
          <Navbar />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
