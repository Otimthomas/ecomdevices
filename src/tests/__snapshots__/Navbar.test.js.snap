// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> matches the snapshot 1`] = `
<nav
  className="sc-bdvvtL lpflGG"
>
  <div
    className="header"
  >
    <p
      className="logo"
    >
      LOGO
    </p>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </li>
    </ul>
  </div>
  <hr />
</nav>
`;
